src/content/resources/cpp/multidimensional-arrays.md

```diff
@@ -0,0 +1,63 @@
+---
+authors:
+  - "alearori#8661"
+created_at: 2019/11/15
+title: "Multidimensional Arrays"
+---
+
+## What is a multidimensional array?
+
+To put simply, a multidimensional array is an array that contains more than one row and column. It is commonly used for making matricies, but it can also be used for storing information such as if you use `string`. The format of the container is `{{a1,a2,a3,...},{b1,b2,b3,...},{c1,c2,c3,...},...}`, where `a1`,`a2`,`a3`, and etc. are objects of the data type that was array was declared to contain at the start.
```

Comment:
Maybe describe it as "an array of arrays"? The row/column stuff might be a little confusing

Reply:
I changed the row/column to fit the "layers" analogy above, and added more about it.